docs/network-flow-visibility.md

```diff
@@ -24,18 +24,22 @@
     - [Correlation of Flow Records](#correlation-of-flow-records)
     - [Aggregation of Flow Records](#aggregation-of-flow-records)
 - [Quick deployment](#quick-deployment)
-- [ELK Flow Collector](#elk-flow-collector)
-  - [Purpose](#purpose)
-  - [About Elastic Stack](#about-elastic-stack)
-  - [Deployment Steps](#deployment-steps)
-  - [Pre-built Dashboards](#pre-built-dashboards)
-    - [Overview](#overview-1)
-    - [Pod-to-Pod Flows](#pod-to-pod-flows)
-    - [Pod-to-External Flows](#pod-to-external-flows)
-    - [Pod-to-Service Flows](#pod-to-service-flows)
-    - [Flow Records](#flow-records)
-    - [Node Throughput](#node-throughput)
-    - [Network Policy](#network-policy)
+- [Flow Collector](#flow-collector)
+  - [IPFIX Collector](#ipfix-collector)
+    - [Deployment Steps](#deployment-steps)
+    - [Ouput Flow Records](#output-flow-records)
+  - [ELK Flow Collector](#elk-flow-collector)
+    - [Purpose](#purpose)
+    - [About Elastic Stack](#about-elastic-stack)
+    - [Deployment Steps](#deployment-steps-1)
+    - [Pre-built Dashboards](#pre-built-dashboards)
+      - [Overview](#overview-1)
+      - [Pod-to-Pod Flows](#pod-to-pod-flows)
+      - [Pod-to-External Flows](#pod-to-external-flows)
+      - [Pod-to-Service Flows](#pod-to-service-flows)
+      - [Flow Records](#flow-records)
+      - [Node Throughput](#node-throughput)
+      - [Network Policy](#network-policy)
```

Comment:
@zyiou Do you think we need to keep these in the table of contents? Now they are in heading level 5, which makes the fonts smaller than the body texts. If we can remove them from the table of contents, they can be formatted as item lists under `Pre-built Dashboards`.

Reply:
I'm fine with current toc layout.